### PR TITLE
Awsome navigation

### PR DESCRIPTION
**Pull request by MohamedRoshdy2021**
in this PR we have 

- Added the SPA approach to our app like displaying and removing elements only in the dom
- Added the contact-Form and the navbar
- improved the styling of the app

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MohamedRoshdy2021, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- SPA without library ✔️
- UI according to wireframe ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] Your MIT link goes to a 404 page, kindly fix it.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -8,22 +8,44 @@
   <title>Awsome Books</title>
 </head>
 <body>
+  <nav>
+    <h2>Awesome Books</h2>
+    <ul>
+      <li id="list">List</li>
```
Comment (shella12) [inline index.html:14]: - [x] [OPTIONAL] Please make sure after making all the new changes your add book function is still working as currently when I try to add book I see following error and no book is showing in the list 

Screenshot 2023-07-05 at 4 46 48 PM

**Review by shella12 — APPROVED** [review]
Hi @agneta1991 and @MohamedRoshdy2021, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- License link has been fixed ✔️  
- Implemented all required changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MohamedRoshdy2021) [reply index.html:14]: It's extraordinary that I double-checked everything, and it's working fine for me and my partner !! 
@shella12

**Pull request merged**